core/src/test/java/org/testcontainers/junit/DockerComposeContainerWithBuildTest.java

```diff
@@ -0,0 +1,20 @@
+package org.testcontainers.junit;
+
+import java.io.File;
+import org.junit.Rule;
+import org.testcontainers.containers.DockerComposeContainer;
+
+public class DockerComposeContainerWithBuildTest extends BaseDockerComposeTest {
```

Comment:
I am afraid this test does not test it, and only covers the setter. If we break `withBuild`, the test will continue to pass.

Also, the used compose file (https://github.com/testcontainers/testcontainers-java/blob/151b93ac065a420d0a803d2ebb8ba6d152c783a4/core/src/test/resources/compose-test.yml ) is rather heavy (mysql)

Reply:
Regarding the test I was working on a version which tests the created image and so on. 
Due to time issues I was not bale to finish it.

I can use a lighter compose file.

Reply:
I've taken this on and am pushing the proposed change now.